connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskThreadedTest.java

```diff
@@ -75,6 +77,7 @@
     // These are fixed to keep this code simpler. In this example we assume byte[] raw values
     // with mix of integer/string in Connect
     private static final String TOPIC = "test";
+    private static final Headers EMPTY_HEADERS = new RecordHeaders();
```

Comment:
Again, strictly speaking this is not an immutable object, and it's possible that a test alters the object by adding more headers and this corrupts/affects other tests.

As mentioned above, it might be worth just creating new instances as needed, perhaps with helper methods such as `emptyHeaders()`.

Reply:
Refactored to use `emptyHeaders()`